resources/charts/fork-observer/values.yaml

```diff
@@ -0,0 +1,133 @@
+# Default values for fork-observer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+namespace: warnet
+
+restartPolicy: Always
+
+image:
+  repository: b10c/fork-observer
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podLabels:
+  app: "warnet"
+  mission: "observer"
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+
+livenessProbe:
+  exec:
+    command:
+    - pidof
+    - fork-observer
+  failureThreshold: 3
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  successThreshold: 1
+  timeoutSeconds: 1
+readinessProbe:
+  failureThreshold: 1
+  periodSeconds: 1
+  successThreshold: 1
+  tcpSocket:
+    port: 2323
+  timeoutSeconds: 1
+
+
+# Additional volumes on the output Deployment definition.
+volumes: []
+# - name: foo
+#   secret:
+#     secretName: mysecret
+#     optional: false
+
+# Additional volumeMounts on the output Deployment definition.
+volumeMounts: []
+# - name: foo
+#   mountPath: "/etc/foo"
+#   readOnly: true
+
+port: 2323
+
+# This should be copied to /app/config.toml
+baseConfig: |
+  # fork-observer base configuration file
+
+  # Database path of the key value store. Will be created if non-existing.
+  database_path = "db"
+
+  # path to the location of the static www files
+  www_path = "./www"
+
+  # Interval for checking for new blocks
+  query_interval = 10
```

Comment:
How many nodes do you plan to run? Checking 100 nodes every 10 seconds might be a bit much, but checking 10 nodes every 10 seconds should be fine.

Reply:
Same comment as https://github.com/bitcoin-dev-project/warnet/pull/507#discussion_r1741882126, this should be configurable by the user. It would also allow us to easily redeploy fork-observer with different intervals if we find its starting to slow down for a particular network.